fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexerBuildValueIndexTest.java

```diff
@@ -166,7 +166,6 @@ private void valueRebuild(@Nonnull List<TestRecords1Proto.MySimpleRecord> record
         valueRebuild(records, null);
     }
 
-
```

Comment:
I think it should be possible to write a test to actually verify that the safe build is doing what it promises. I think you could do this by handing on `OnlineIndexBuilder` a custom `FDBDatabaseRunner` that has a lock that it acquires before starting the next transaction.  In this fashion, you could have it do a little bit of work, then wait to acquire the lock, then do the next piece of work.   The test harness could then own this lock, and after the first bit of work is complete, it could attempt to start another build, release the lock and see what happens when the two of them are running at the same time. 

Similarly, you could do something like that but allow the lease to expire so that the second run finishes the index build (or restarts it).